src/components/molecules/VenueCard/VenueCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import * as S from "./VenueCard.styles";
```

Comment:
[NB, potentially OOS] Would be good to git rid of `styled-components` here when we can.

Reply:
I agree, this is being used in the last tab, which is the legacy page from admin v2, this last tab will be changed to "Run" from admin v3 designs and all of the styled components will be removed.